application/testing/CMakeLists.txt

```diff
@@ -1049,6 +1049,9 @@ f3d_test(NAME TestConfigFileQuiet DATA nonExistentFile.vtp CONFIG ${F3D_SOURCE_D
 # Test no file with config file
 f3d_test(NAME TestNoFileConfigFile CONFIG ${F3D_SOURCE_DIR}/testing/configs/verbose.json ARGS --verbose REGEXP "No files to load provided" NO_BASELINE)
 
+# Test that --no-config set no-config value to 1
+f3d_test(NAME TestNoConfigWithConfig ARGS --no-config --config=config_build --verbose REGEXP "'no-config' = '1'" NO_BASELINE)
```

Comment:
instead of testing the verbosity of no-config, how about opening a file a taking a screenshot ? This way we ensure the config_build is not taken into account

Reply:
Make sense.

Reply:
I am able to create the image:

```bash
bin/f3d \
  --output testing/baselines/TestNoConfigWithConfig.png \
  --no-config \
  --resolution=300,300 \
  --config testing/configs/complex.json \
  testing/data/f3d.glb
```

But I do not understand why the test does not work.

```patch
- f3d_test(NAME TestNoConfigWithConfig ARGS --no-config --config=config_build --verbose REGEXP "'no-config' = '1'" NO_BASELINE)
+ #### bin/f3d --output a.png --no-config --config testing/configs/complex.json
+ #### DATA f3d.glb
+ #### CONFIG ${F3D_SOURCE_DIR}/testing/configs/complex.json
+ #### UI
+ #### f3d_test(NAME TestInteractionConsoleOpen DATA f3d.glb INTERACTION UI) #Escape
+
+ #### f3d_test(NAME TestNoConfigWithConfig DATA f3d.glb ARGS --no-config --config ${F3D_SOURCE_DIR}/testing/configs/complex.json UI)
+ #### f3d_test(NAME TestInteractionConsoleOpen DATA f3d.glb INTERACTION UI) #Escape
+ f3d_test(NAME TestNoConfigWithConfig DATA f3d.glb UI NO_BASELINE)
```

I don't know how to create a test that compare the Temporary with the baseline image.

Reply:
Here is how to do it:

1. Remove current test and instead add:
`f3d_test(NAME TestNoConfigWithConfig DATA f3d.glb ARGS --no-config --config config_build)`
2. configure, build and run `ctest -R TestNoConfigWithConfig` locally, test will fail as expected
3. Recover the baseline from `Testing/Temporary/TestNoConfigWithConfig.png`, check it is correct and copy it to `src/testing/baselines/`
4. Commit and push

Reply:
Thanks.